<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>464</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>464</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>464</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>JobSearch</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>461</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Home">
     <property name="maximumSize">
      <size>
       <width>470</width>
       <height>464</height>
      </size>
     </property>
     <widget class="QFrame" name="frame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>431</width>
        <height>391</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #eea;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="txtJob">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>281</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <stylestrategy>PreferAntialias</stylestrategy>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
       <property name="placeholderText">
        <string>عنوان شغلی، مهارت و...</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cmCity">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>160</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <stylestrategy>PreferAntialias</stylestrategy>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
       <item>
        <property name="text">
         <string>تهران</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>اصفهان</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>یزد</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>خراسان رضوی</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>فارس</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="cmCategory">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>220</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <stylestrategy>PreferAntialias</stylestrategy>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
       <item>
        <property name="text">
         <string>همه‌ی دسته‌بندی‌ها</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>فروش و بازاریابی</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>وب،‌ برنامه‌نویسی و نرم‌افزار</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>مالی و حسابداری</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>مسئول دفتر، اجرائی و اداری</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>411</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>17</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <stylestrategy>PreferAntialias</stylestrategy>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;
</string>
       </property>
       <property name="text">
        <string>به جاب سرچ خوش آمدید</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSearch">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>290</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: green;
color:white;
border-radius:60%;</string>
       </property>
       <property name="text">
        <string>جستجو</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnResult">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>350</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: red;
color:white;
border-radius:60%;</string>
       </property>
       <property name="text">
        <string>نتیجه</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtUsername">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>120</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <stylestrategy>PreferAntialias</stylestrategy>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
       <property name="placeholderText">
        <string>ایمیل</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtPassword">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <stylestrategy>PreferAntialias</stylestrategy>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
       <property name="placeholderText">
        <string>رمزعبور</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnExit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: red;
color:white;
border-radius:60%;</string>
       </property>
       <property name="text">
        <string>خروج</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Result">
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>431</width>
        <height>391</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #eea;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>10</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <stylestrategy>PreferAntialias</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>نتیجه:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnHome">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>350</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: red;
color:white;
border-radius:60%;</string>
       </property>
       <property name="text">
        <string>صفحه قبل</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtResult">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>411</width>
         <height>301</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <stylestrategy>PreferAntialias</stylestrategy>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#eee;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>470</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
